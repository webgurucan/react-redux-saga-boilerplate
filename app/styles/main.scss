// *** Vendor
@import '~sass-mediaqueries/media-queries';

// *** Base
@import 'variables/common';
@import 'variables/bootstrap';
@import 'variables/typography';
@import 'modules/icons';

// *** Utilities
@import 'utilities/functions';
@import 'utilities/mixins';
@import 'utilities/grid';

// *** Vendor
@import 'vendor/bootstrap';

// *** Modules
@import 'modules/animations';
@import 'modules/buttons';
@import 'modules/transitions';

// *** Layout
@import 'layout/global';
@import 'layout/app';

// *** Components
@import 'components/alert';
@import 'components/footer';
@import 'components/header';
@import 'components/loader';
@import 'components/logo';
@import 'components/system-alerts';

// *** Containers
@import 'containers/github';

// *** Routes
@import 'routes/home';
@import 'routes/private';
@import 'routes/not-found';
