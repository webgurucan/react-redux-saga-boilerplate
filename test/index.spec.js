import 'main';
import createRoutes from 'routes';
import { ActionTypes, XHR } from 'constants/index';

describe('Constants:ActionTypes', () => {
  it('should match the snapshot', () => {
    expect(ActionTypes).toMatchSnapshot();
  });
});


describe('Constants:XHR', () => {
  it('should match the snapshot', () => {
    expect(XHR).toMatchSnapshot();
  });
});

describe('Routes', () => {
  it('should match the snapshot', () => {
    const routes = createRoutes();
    expect(routes.props.children.length).toMatchSnapshot();
  });
});
