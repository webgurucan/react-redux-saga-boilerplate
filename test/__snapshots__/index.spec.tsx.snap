// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index/app should have mounted the app 1`] = `
<div>
  <div data-testid="app"
       class="AppWrapper-sc-11ytvmm-0 fBzUHJ"
  >
    <main class="Main-sc-11ytvmm-1 kONFGE">
      <div data-testid="Home"
           class="Background-dd6dxc-0 hdGgTG"
      >
        <div class="sc-jrsJWt ksZcRw">
          <div class="Header-vz2az5-0 bfCkCF">
            <div class="Wrapper-ckpwlu-0 kQwYGM">
              <svg src="/media/brand/icon.svg">
              </svg>
            </div>
          </div>
          <h1 class="Heading-vz2az5-1 iUMFxr">
            React Redux Saga Boilerplate
          </h1>
          <button data-testid="Login"
                  type="button"
                  class="sc-hKFxyN dCNtMJ"
          >
            <svg height="100%"
                 src="/media/icons/sign-in.svg"
                 width="20px"
                 class="IconWrapper-oqx6ge-0 irLJro"
            >
            </svg>
            <span class="sc-ksluID ihgDFX">
              Start
            </span>
          </button>
        </div>
      </div>
    </main>
    <footer class="sc-bdnxRM eMiFrX">
      <div class="sc-jrsJWt hbVsEW">
        <div class="sc-iqAclL iDusPv">
          <iframe frameborder="0"
                  height="20px"
                  scrolling="0"
                  src="https://ghbtns.com/github-btn.html?user=gilbarbara&amp;repo=react-redux-saga-boilerplate&amp;type=star&amp;count=true"
                  title="GitHub Stars"
                  width="110px"
          >
          </iframe>
          <iframe frameborder="0"
                  height="20px"
                  scrolling="0"
                  src="https://ghbtns.com/github-btn.html?user=gilbarbara&amp;type=follow&amp;count=true"
                  title="GitHub Follow"
                  width="130px"
          >
          </iframe>
        </div>
      </div>
    </footer>
    <div class="SystemAlertsWrapper-sc-1eg1hzx-5 beGXka">
      <div class="Base-sc-1eg1hzx-0 TopLeft-sc-1eg1hzx-1 jElLEB lbzRu">
        <div>
        </div>
      </div>
      <div class="Base-sc-1eg1hzx-0 TopRight-sc-1eg1hzx-2 jElLEB jjENDe">
        <div>
        </div>
      </div>
      <div class="Base-sc-1eg1hzx-0 BottomLeft-sc-1eg1hzx-3 jElLEB euXyyg">
        <div>
        </div>
      </div>
      <div class="Base-sc-1eg1hzx-0 BottomRight-sc-1eg1hzx-4 jElLEB jygNZo">
        <div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
